spec/requests/admin/split_details_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe Admin::SplitsController do
+  include Warden::Test::Helpers
+
+  describe 'GET /admin' do
+    let(:default_app) { FactoryBot.create(:app, name: "default_app", auth_secret: "6Sd6T7T6Q8hKcoo0t8CTzV0IdN1EEHqXB2Ig4raZsOf") }
```

Comment:
Do the tests pass without the `auth_secret` value passed? I'm wondering if this (fake, definitely fake! lol) auth secret is only necessary for API-based tests.

The `login_as` below should do the real auth for these requests.